projects/chaos/src/gpio_seq.c

```diff
@@ -0,0 +1,39 @@
+#include "gpio_seq.h"
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "delay.h"
+#include "gpio.h"
+#include "status.h"
+
+static uint32_t s_delay_us;
+
+void gpio_seq_set_delay(uint32_t delay_us) {
+  s_delay_us = delay_us;
+}
+
+uint32_t gpio_seq_get_delay(void) {
+  return s_delay_us;
+}
+
+StatusCode gpio_seq_init_pins(const GPIOAddress *addrs, size_t n_addrs,
```

Comment:
I think you might as well make this `num_addrs` :)

Reply:
Done.